physionet-django/project/modelcomponents/access.py

```
@@ -167,6 +168,35 @@ class Meta:
         default_permissions = ()
 
 
+class DataSource(models.Model):
+    """
+    Controls all access to project data.
+    """
+    class DataLocation(models.TextChoices):
+        DIRECT = 'DI', _('Direct')
+        GOOGLE_BIGQUERY = 'GBQ', _('Google BigQuery')
+        GOOGLE_CLOUD_STORAGE = 'GCS', _('Google Cloud Storage')
+        AWS_OPEN_DATA = 'AOD', _('AWS Open Data')
+        AWS_S3 = 'AS3', _('AWS S3')
+
+    class AccessMechanism(models.TextChoices):
+        GOOGLE_GROUP_EMAIL = 'google-group-email', _('Google Group Email')
+        S3 = 's3', _('S3')
+        RESEARCH_ENVIRONMENT = 'research-environment', _('Research Environment')
+
+    project = models.ForeignKey('project.PublishedProject',
+                                related_name='data_sources', db_index=True, on_delete=models.CASCADE)
+    files_available = models.BooleanField(default=False)
+    data_location = models.CharField(max_length=3, choices=DataLocation.choices, default=DataLocation.DIRECT)
```

Aren't the default values here a bit misleading?